packages/mobile/ios/Podfile.lock

```diff
@@ -389,8 +392,6 @@ PODS:
     - React
   - RNPermissions (2.0.2):
     - React
-  - RNRandomBytes (3.0.0):
```

Comment:
Do you know why this is being removed?

Reply:
No but we don't seem to use it. Not sure why my cocopods removed it when Jean's didn't